WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/ReattachUploadingMediaUseCase.kt

```diff
@@ -0,0 +1,41 @@
+package org.wordpress.android.ui.posts.editor.media
+
+import org.wordpress.android.editor.EditorMediaUploadListener
+import org.wordpress.android.fluxc.model.MediaModel
+import org.wordpress.android.ui.posts.EditPostRepository
+import org.wordpress.android.ui.uploads.UploadServiceFacade
+import javax.inject.Inject
+
+class ReattachUploadingMediaUseCase @Inject constructor(
+    private val editPostRepository: EditPostRepository,
```

Comment:
I don't think this will work because the injected edit post repository is not initialized with the post. What we do in the `EditPostActivity` is that we inject the repository and load the post in `onCreate`. We do not share the same instance of the repository because there might be more instances of the activity. I think we have to pass the repository as a parameter in `reattachUploadingMediaForAztec`.

Reply:
Oh, my bad. You are right. I'm wondering now, what the code actually does - because I tested the functionality after the changes and it worked :D.